config/base/webpack.config.ts

```diff
@@ -8,7 +8,7 @@ import Config from "webpack-config";
 
```

Comment:
this entire file is unused except as an import to test webpack config, in the future it will be used for more advanced builds on this project